.gitignore

```diff
@@ -0,0 +1,13 @@
+# Logs
+logs
+*.log
+yarn-debug.log*
+yarn-error.log*
+
+# Diagnostic reports (https://nodejs.org/api/report.html)
+report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
+
```

Comment:
Would also add `.DS_Store` which is generated by mac and I I've seen it already in few PRs accidentally

Reply:
That's fair, I haven't used finder to look at the internals of a repo in a long while. Easy add ➕